.gitlab-ci.yml

```diff
@@ -1,5 +1,5 @@
 default:
-  image: img.doom.fm/build/golang:latest-1.15.3-buster
+  image: img.doom.fm/build/golang:latest-1.18.7-buster
```

Comment:
```suggestion
  image: reg.agarg.me/build/golang:latest-1.18.7-bullseye
```

Reply:
The old registry may or may not exist, but this is the current one I’d like to use.